tests/api/test_statements_post.py

```diff
@@ -0,0 +1,224 @@
+"""
+Tests for the POST statements endpoint of the Ralph API.
+"""
+import re
+from unittest import mock
+from uuid import uuid4
+
+from elasticsearch import Elasticsearch
+from elasticsearch.client import IndicesClient
+from fastapi.testclient import TestClient
+
+from ralph.api import app
+
+from tests.fixtures.backends import ES_TEST_HOSTS, ES_TEST_INDEX
+
+ES_CLIENT = Elasticsearch(ES_TEST_HOSTS)
+ES_INDICES_CLIENT = IndicesClient(ES_CLIENT)
+
+client = TestClient(app)
+
+
+# pylint: disable=invalid-name,unused-argument
+@mock.patch("ralph.api.routers.statements.ES_INDEX", ES_TEST_INDEX)
+def test_post_single_statement_directly(auth_credentials, es):
+    """
+    Statements POST supports adding one statement by passing it as a JSON dict.
+    """
+
+    statement = {
+        "actor": {
+            "account": {
+                "homePage": "https://example.com/homepage/",
+                "name": str(uuid4()),
+            },
+            "objectType": "Agent",
+        },
+        "id": str(uuid4()),
+        "object": {"id": "https://example.com/object-id/1/"},
+        "timestamp": 1647349671,
+        "verb": {"id": "https://example.com/verb-id/1/"},
+    }
+
+    response = client.post(
+        "/xAPI/statements/",
+        headers={"Authorization": f"Basic {auth_credentials}"},
+        json=statement,
+    )
+
+    assert response.status_code == 200
+    assert response.json() == [statement["id"]]
+    ES_INDICES_CLIENT.refresh()
+    hits = ES_CLIENT.search(index=ES_TEST_INDEX)["hits"]["hits"]
```

Comment:
The `es` fixture is an elasticsearch client instance, so you can use it as follow:

```suggestion
    es.indices.refresh(index=ES_TEST_INDEX)
    hits = es.search(index=ES_TEST_INDEX)["hits"]["hits"]
```

Reply:
The same pattern applies for subsequent tests.